template/LICENSE

```diff
@@ -0,0 +1,20 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
Where did you take it? Why not https://github.com/tqtezos/baseDAO/blob/master/LICENSE?

Reply:
Don't we want people to freely use this without any restrictions? As we do for e.g. config file templates

Reply:
I had the same thought but ended up with conclusion that MIT is very permissive anyway. The difference only matters for those who want to apply a different license here AFAIU. And I am not sure if we want to permit that. On the one hand, Unlicense should decrease usage barrier which is good for us. On the other hand, if someone writes their own cool DAO, we probably want them to keep our copyright notice and make their cool DAO public and available for everybody. Though probably if someone wants to use a different license, they can create a new project without our templates. So I don't know what's better, both options seem acceptable to me now. However, even if you decided to keep Unlicense, please update `SPDX-FileCopyrightText` in some files to TQ, it still seems more correct than Serokell.